legend-engine-application-query/src/main/java/org/finos/legend/engine/application/query/configuration/ApplicationQueryConfiguration.java

```diff
@@ -17,10 +17,25 @@
 import com.mongodb.client.MongoClient;
 import com.mongodb.client.MongoClients;
 import org.finos.legend.engine.shared.core.vault.Vault;
+import redis.clients.jedis.DefaultJedisClientConfig;
+import redis.clients.jedis.HostAndPort;
+import redis.clients.jedis.JedisPooled;
 
 public class ApplicationQueryConfiguration
 {
-    public static MongoClient getMongoClient()
+    public static Object getQueryStoreClient()
```

Comment:
Lets avoid Object, and return QueryStoreManager.

Reply:
Hi @rafaelbey, I removed the use of Object but it required me to push the conditional configuration logic, which decides between Redis and MongoDB, to the Server layer. As I did that I noticed I missed committing Redis client dependencies so those are now included to. This is ready for your review. Thanks, Allen